crates/bevy_time/src/lib.rs

```diff
@@ -84,20 +85,15 @@ pub enum TimeUpdateStrategy {
     ManualDuration(Duration),
 }
 
-/// Channel resource used to receive time from the render world.
-#[derive(Resource)]
-pub struct TimeReceiver(pub Receiver<Instant>);
-
-/// Channel resource used to send time from the render world.
-#[derive(Resource)]
-pub struct TimeSender(pub Sender<Instant>);
+/// Queue resource used to receive time from the render world.
+#[derive(Resource, Clone)]
+pub struct TimeEventQueue(pub Arc<ConcurrentQueue<Instant>>);
```

Comment:
Seeing the transition from "channel" -> `Arc<ConcurrentQueue>` does incline me to echo cart's concerns about rocking the boat. It's probably still worth it just to remove a spinny dependency though.

Should probably have a different name though since this doesn't wrap the `EventQueue` type